package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.68.8",
```

Comment:
The changes in `package-lock.json` aren't needed, and it would be really great if you could restore this file to the state the `main` branch's `package-lock.json` is.

You could use the following commands to do the above -
`git remote add upstream https://github.com/EddieHubCommunity/LinkFree.git`
`git fetch upstream`
`git checkout main`
`git reset --hard upstream/main`
`git checkout linkflash`
`git checkout main -- package-lock.json`
`git add package-lock.json`
`git commit`
`git push`

Reply:
I didn't make any changes to this file. Just npm install and committing it. I have replaced my packaged-lock.json with what is available on the main branch. Hope it's okay

Reply:
You shouldn't commit it, you should only commit the files you actually worked on. The changes need to be reverted before this PR can be merged.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
I see you managed to revert the changes, great! (sorry, I didn't look before my previous comment)

Reply:
Alright, no problem.